include/types.h

```diff
@@ -348,7 +426,7 @@ struct MarioState
     /*0x68*/ struct Surface *floor;
     /*0x6C*/ f32 ceilHeight;
     /*0x70*/ f32 floorHeight;
-    /*0x74*/ s16 floorAngle;
+    /*0x74*/ s16 floorYaw;
```

Comment:
please revert this change. could cause a lot of conflicts

Reply:
waiting for testing before resolving

Reply:
i agree, this sort of renaming doesn't really add anything, it just creates conflicts